aviatrix/utils.go

```diff
@@ -196,3 +200,97 @@ func DiffSuppressFuncGatewayVpcId(k, old, new string, d *schema.ResourceData) bo
 
 	return false
 }
+
+func reverseArray(arr []string) []string {
+	for i, j := 0, len(arr)-1; i < j; i, j = i+1, j-1 {
+		arr[i], arr[j] = arr[j], arr[i]
+	}
+	return arr
+}
+
+// sort the firewall_image_version list
```

Comment:
Comments about the function should start with the function name. So in this case it might be:
`sortVersion sorts the firewall_image_version list`

Reply:
have fixed, thank you